tools/newgtlds.go

```diff
@@ -22,6 +22,10 @@ const (
 	// 2). See https://www.icann.org/resources/pages/registries/registries-en for
 	// more information.
 	ICANN_GTLD_JSON_URL = "https://www.icann.org/resources/registries/gtlds/v2/gtlds.json"
+	// IANA_TLDS_TXT_URL is the URL for the IANA "Public Suffix List" of TLDs 
+	// in the ICP-3 Root - including new ccTLDs, EBRERO gTLDS or things not in
+	// the JSON File above that should be included inthe PSL.  Note: UPPERCASE
```

Comment:
```suggestion
	// the JSON File above that should be included in the PSL.  Note: UPPERCASE
```

Reply:
&nbsp; | **Delegated TLD (g or cc)** | **Contracted, PreDelegation** | **Cancelled, Delegated** | **Cancelled, PreDelegation** | **IETF/Special** | **Delegated ccTLD** | **Delegated IDN ccTLD**
-- | -- | -- | -- | -- | -- | -- | --
**Entry in last PSL** | YES | EITHER, See Note | YES, See Note | YES, See Note | EITHER, See Note | NO | NO
**Entry in ICANN JSON** | YES | YES | YES | YES | EITHER | N/A | N/A
**JSON : Cancelled?** | No | No | YES | YES | NO | N/A | N/A
**In IANA TLD TXT** | YES | &nbsp; | YES | No | YES | YES | YES
&nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp;
**Notes** | Status Quo | If missing, Add Entry | Currently:  Remove from PSL;  Revise to:  Leave in PSL, change commented line | Remove from PSL | If missing, Add Entry | If missing, Add Entry | If missing, Add Entry - Need to convert from punycode to UTF-8 and add that
&nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp; | &nbsp;
**Comment Line Change:** | &nbsp; | &nbsp; | ```// [tld] : [Date Cancelled]; EBERO TLD Active``` | &nbsp; | ```// [tld] : [run date]``` | ```// [tld] : [run date]``` | ```// [A-tld] : [run date]``` (note [A-tld] is the A-Label/Punycode)